run-app.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+run_app() {
```

Comment:
Just FYI I will probably put the A/B setup in this file after this lands... Let me know if that makes sense.